spec/features/applications/can_see_pet_names_on_favorites_page_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
```

Comment:
Can probably just included this test on the favorites show page test. Will make note to refactor